protected/humhub/modules/stream/actions/Stream.php

```diff
@@ -433,4 +432,22 @@ public function getStreamQuery()
     {
         return $this->streamQuery;
     }
+
+    /**
+     * @return bool
+     */
+    public function isSingleContentRequest()
```

Comment:
@yurabakhtin Can we make the method private until it is otherwise used?

Reply:
@luke- Ok, done https://github.com/humhub/humhub/pull/4986/commits/8ce93c0e5bab1e1f6a687b0dc89f5bbed00240a4.